pkg/version/version.go

```diff
@@ -74,6 +70,17 @@ func LatestReleaseInDir(releasesDir string) (string, error) {
 		}
 	}
 
+	// If we reach this point and the highestVersion is still at v0.0.0 (which is
+	// what it was initialized at), then we didn't find any valid release files in
+	// the specified dir.
+	if highestVersion.Equal(*semver.New("0.0.0")) {
```

Comment:
This logic doesn't have a corresponding test in `version_test.go` afaict

Reply:
added "TestLatestReleaseInDirNoValidFiles"